src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -223,9 +221,7 @@ public AgentSettings LoadSettings()
                 agentName = consoleWizard.ReadValue(CliArgs.Agent,
                                                 StringUtil.Loc("AgentName"),
                                                 false,
-                                                // TODO: coreCLR doesn't expose till very recently (Jan 15)
-                                                // Environment.MachineName,
-                                                "myagent",
+                                                Environment.MachineName,
```

Comment:
Ok, I went over the code.. we handle agent name being null or empty. We wont break anything.. When its null, we wont show the default value.

However, I changed the code to 
`Environment.MachineName ?? "myagent",`